x-pack/plugins/security_solution/public/common/components/event_details/insights/related_alerts_upsell.tsx

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React from 'react';
+import { EuiFlexGroup, EuiFlexItem, EuiIcon, EuiText } from '@elastic/eui';
+
+import { euiStyled } from '@kbn/kibana-react-plugin/common';
+import { ALERT_UPSELL } from './translations';
+
+const UpsellContainer = euiStyled.a`
+  border: 1px solid ${({ theme }) => theme.eui.euiColorLightShade};
+  padding: 12px;
+  border-radius: 6px;
+  color: ${({ theme }) => theme.eui.euiTextSubduedColor};
+
+  &:focus, &:hover {
+    text-decoration: underline;
+  }
+`;
+
+const StyledIcon = euiStyled(EuiIcon)`
+  margin-right: 10px;
+`;
+
+export const RelatedAlertsUpsell = React.memo(() => {
+  return (
+    <UpsellContainer href="https://www.elastic.co/pricing/" target="_blank">
+      <EuiFlexGroup alignItems="center" gutterSize="none">
+        <EuiFlexItem grow={false}>
+          <StyledIcon size="m" type="lock" />
+        </EuiFlexItem>
+        <EuiFlexItem>
+          <EuiText size="s">{ALERT_UPSELL}</EuiText>
+        </EuiFlexItem>
+      </EuiFlexGroup>
+    </UpsellContainer>
```

Comment:
Consider the following suggestion that:

- Replaces the `href` and `target` props to `UpsellContainer` with an `EuiLink`
- `EuiLink` is given `rel="noopener noreferrer"` as a standard practice for external links to both:
  - prevent [tabnabbing](https://cheatsheetseries.owasp.org/cheatsheets/HTML5_Security_Cheat_Sheet.html#tabnabbing) (unlikely given the source we're linking to)
  - prevent information leakage via the `Referer` HTTP request header
- `EuiLink` uses the `external={true}` prop, which renders an external link indicator icon

```suggestion
    <UpsellContainer>
      <EuiFlexGroup alignItems="center" gutterSize="none">
        <EuiFlexItem grow={false}>
          <StyledIcon size="m" type="lock" />
        </EuiFlexItem>
        <EuiFlexItem>
          <EuiText size="s">
            <EuiLink
              color="subdued"
              external={true}
              href="https://www.elastic.co/pricing/"
              rel="noopener noreferrer"
              target="_blank"
            >
              {ALERT_UPSELL}
            </EuiLink>
          </EuiText>
        </EuiFlexItem>
      </EuiFlexGroup>
    </UpsellContainer>
```

If the suggestion above is implemented:

- The following styles in `UpsellContianer` are no longer required, because they are built into `EuiLink`:

```
  &:focus, &:hover {
    text-decoration: underline;
  }
```

- An external link icon will appear next to the link, per the before / after screenshots below:

**Before**

![with_styled_panel](https://user-images.githubusercontent.com/4459398/183761413-2e838884-6434-4dfb-86d6-af72d1a95b9a.png)

**After**

![with_eui_link](https://user-images.githubusercontent.com/4459398/183761460-c391d2d0-601c-4d6e-8ea9-652f75c739ce.png)

Reply:
Oh nice, I was not thinking about using an `EuiLink`, good to know that it has all the default styling and the correct referrer handling! https://github.com/elastic/kibana/pull/138419/commits/681b463a5b926b49f067a0d1436a1570233d629a